src/components/DateInput.jsx

```diff
@@ -29,6 +29,7 @@ const propTypes = forbidExtraProps({
   placeholder: PropTypes.string, // also used as label
   displayValue: PropTypes.string,
   screenReaderMessage: PropTypes.string,
+  ariaLabelledBy: PropTypes.string,
```

Comment:
Suggestion: It might be worth creating a custom prop type here that supports a valid id type that's space separated. 

In a separate project, I created this regex for a single id: 
```const idRegex = /^_|^-|^\./;```

And you'd want to apply that for space separated values so that multiple ids can be passed.

Info on valid id values: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

Reply:
Great idea!